templates/experiments/components/experiment_chat.html

```diff
@@ -13,6 +13,9 @@
             {% if perms.annotations.add_customtaggeditem %}
               <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold">Tags</th>
             {% endif %}
+            {% if perms.annotations.add_customtaggeditem %}
```

Comment:
This should only require a view permission since the user can't add / remove them:
```suggestion
            {% if perms.annotations.view_customtaggeditem %}
```

Reply:
[3945761](https://github.com/dimagi/open-chat-studio/pull/440/commits/39457616235039747b54db29804b02d7062afc05)